llvm/tools/llvm-dwp/Opts.td

```diff
@@ -9,5 +9,7 @@ def version : F<"version", "Display the version of this program">;
 
 def execFileNames : S<"e", "Specify the executable/library files to get the list of *.dwo from.">, MetaVarName<"<filename>">;
 def outputFileName : S<"o", "Specify the output file.">, MetaVarName<"<filename>">;
-def continueOnCuIndexOverflow: F<"continue-on-cu-index-overflow", "This turns an error when offset for .debug_*.dwo sections "
-                                         "overfolws into a warning.">, MetaVarName<"<filename>">;
+def continueOnCuIndexOverflow : S<"continue-on-cu-index-overflow", " =soft-stop, This produces a truncated but valid "
```

Comment:
Maybe make "continue" as the default?

Reply:
Now:

- When this option is not given, the operation executes a 'hard-stop'.
- When this option  is given but no values specified, it executes 'continues' and continues to generate a DWP file with overflowed offset information..
- When its value= 'soft-stop', it executes 'continues' and generates a 'best effort' DWP file.
- Else: When its value is others, it also executes 'continues'